test/unit/api/cpp/term_black.cpp

```diff
@@ -1208,6 +1208,15 @@ TEST_F(TestApiBlackTerm, getRealAlgebraicNumber)
   }
 }
 
+TEST_F(TestApiBlackTerm, getSkolem)
+{
+  // ordinary variables are not skolems
+  Term x = d_tm.mkConst(d_tm.getIntegerSort(), "x");
```

Comment:
Currently we don't allow users to create skolems, do we? This is why this test seems trivial.

Reply:
Yes, creation of skolems will be done in a followup PR, which will make it easy to add new unit tests.